cosmo_tester/test_suites/ha/ha_negative_test.py

```diff
@@ -26,21 +26,21 @@
 
 
 @pytest.fixture(scope='function')
-def cluster(
+def hosts(
```

Comment:
this could've been left named "cluster" since it actually is a cluster :)

Reply:
True :D I think consistency with the rest of the tests makes sense though, and also it means if someone searches for "cluster" in this file they'll find the references they probably actually care about